community/examples/AMD/README.md

```diff
@@ -1,5 +1,9 @@
 # AMD solutions for the HPC Toolkit
 
+> [!NOTE]
+> This document uses SlurmGCP v5 version of hcls blueprint. If you want to
+> use SlurmGCP v6 version, please refer [blueprint](./hpc-amd-slurm-v6.yaml).
```

Comment:
This text needs a bit of cleaning up. Let's try

```
> This document uses Slurm-GCP v5. A newer [blueprint](./hpc-amd-slurm-v6.yaml) using v6
> has been published. This documentation will be updated and the v5 blueprint deprecated.
```

Reply:
Updated.